### DISCUSSION TITLE
Cannot access Teleport via Github

### DISCUSSION THREAD

**Discussion by Chuyentest123** [gravitational/teleport#27263] [Q&A]
Hope you answer my issue soon.
I just do as the document but I can't access Teleport via GitHub
Here is my config file at Teleport and GitHub.yaml

![image](https://github.com/gravitational/teleport/assets/128662730/37156a66-2681-4062-8867-3191471cccd5)

![image](https://github.com/gravitational/teleport/assets/128662730/1d72a7d1-b536-41a3-93e0-6a53185c6533)

And here is my organization and my Team:
![image](https://github.com/gravitational/teleport/assets/128662730/a535cd60-9faa-4020-870a-87b7e8d0459f)

![image](https://github.com/gravitational/teleport/assets/128662730/71b6f28d-a657-4ddb-8010-78ec55a96715)

And the OAuthApps:
![image](https://github.com/gravitational/teleport/assets/128662730/ddfc7555-3049-4a60-8f30-776238a11a2b)

Here is the progress of login:

1. This is Chuyentest123's GitHub account (Administrator of the organization and member of TestEditor):
![image](https://github.com/gravitational/teleport/assets/128662730/8b1ebd48-fe55-49ab-b001-a22eee9c735e)

2. After login GitHub success I have the notification like this (I could not take the photo of Chuyentest123 so I took this photo instead) :
![image](https://github.com/gravitational/teleport/assets/128662730/0bdc782a-ebc2-4d3e-a5dd-0ca8ead27adb)

And after clicking authorize my-user, it failed:
![image](https://github.com/gravitational/teleport/assets/128662730/503c866f-4296-46aa-8273-7c7f8c12b2b5)

Here is the log of the failed login:
![image](https://github.com/gravitational/teleport/assets/128662730/edc34747-8976-4b4d-b927-b1a4eb094d2c)

Teleport version: 11.4.4

**Comment by webvictim** [comment]
From the audit log screenshot, it looks as though the Github user that you're authorising with that email address is only a member of the `TestAuditor` group, not the `TestEditor` group?

![image](https://github.com/gravitational/teleport/assets/897821/fd3c4356-3ce3-4139-baf4-66c8c7bdb281)

**Reply by Chuyentest123** [reply]
No, sorry for the mistake, I need an example of that step so I take the picture from Teleport Guide. ChuyenTest123 is the true account here and it places on both teams. Here is it 
![image](https://github.com/gravitational/teleport/assets/128662730/a8ccd8de-a7de-44be-982a-ca68e880fe8e)

**Comment by webvictim** [comment]
Are you running self-hosted Teleport? If so, please share the Teleport process logs from the Teleport auth and proxy services at the time when you get the "You are not authorized" error. It should give more detail about exactly what the problem is.

If you're using Teleport Team or Enterprise Cloud, please share your tenant name so I can check the logs myself for errors.

**Reply by Chuyentest123** [reply]
Yes, I am. Here is the complete log of this issue. 
![image](https://github.com/gravitational/teleport/assets/128662730/1be73986-8eaa-402e-91b2-965baaee27f5)

**Reply by webvictim** [reply]
I mean the logs from the Teleport process itself - i.e. `sudo journalctl -u teleport`

**Reply by webvictim** [reply]
Although looking at the audit log, it seems like the team names from Github are being lowercased for some reason.

Team names in connectors are likely case-sensitive, so try changing your connector to this:

```yaml
spec:
  teams_to_roles:
  - organization: testTeleport
    team: testeditor
    roles:
    - access
```

**Reply by Chuyentest123** [reply]
![image](https://github.com/gravitational/teleport/assets/128662730/cecb6262-129b-4de6-9877-f3245ee27289)

Hi. I got some work must do three weeks ago so I can't reply to you soon. I'm very sorry about that. Here are the logs you need.

**Reply by Chuyentest123** [reply]
Besides that I have changed the team's name and the organization's name to lowercase but it seems like not working.

**Reply by webvictim** [reply]
This log you shared looks like it's only the top of the log file and not the most recent entries. I'd expect there to be more logs below with details of the Github login attempt. Please try logging in again, then scroll to the bottom of the logs and share that.
